src/udiskslinuxencrypted.c

```diff
@@ -516,6 +519,14 @@ handle_unlock (UDisksEncrypted        *encrypted,
   else {
     if (is_luks)
       name = g_strdup_printf ("luks-%s", udisks_block_get_id_uuid (block));
+    else if (is_bitlk)
+      {
+        uuid = udisks_block_get_id_uuid (block);
+        if (uuid && g_strcmp0 (uuid, "") != 0)
+          name = g_strdup_printf ("bitlk-%s", udisks_block_get_id_uuid (block));
```

Comment:
This should just use `uuid` instead of calling the `get_id_uuid()` function again.

Reply:
Fixed.